scripts/zkai_claim_audit_comparison_artifacts_gate.py

```
@@ -0,0 +1,917 @@
+    if "not_reported" in proof_size_policy and proof_size_comparable:
+        raise ClaimAuditError(f"{row_id} proof size not reported but comparable")
+    if "timing" in require_str(row["primary_metric"], f"{row_id} metric").lower() and "not local" not in timing_policy and "repo_reported" not in timing_policy:
+        raise ClaimAuditError(f"{row_id} timing policy missing source qualifier")
+    if timing_policy in {"", "NA", "unknown"}:
+        raise ClaimAuditError(f"{row_id} timing policy missing")
+    if row_id == "rmsnorm_single_best_label_rejected" and "single best label rejected" not in proof_size_policy:
+        raise ClaimAuditError("favorable-label policy drift")
+    if row_id == "rmsnorm_worst_label_opening_target" and row["primary_value"] != 1401:
+        raise ClaimAuditError("worst-label required reduction drift")
+    if "nanozk" in row_id and not any("NANOZK" in claim for claim in non_claims):
+        raise ClaimAuditError(f"{row_id} NANOZK non-claim missing")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)